process_pr.py

```diff
@@ -1476,11 +1518,11 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
 
     # end of parsing comments section
     cache_comment = None
-    if technical_comment:
-        cache_comment = technical_comment
+    if technical_comments:
+        cache_comment = technical_comments
     else:
```

Comment:
i think this part is not needed any more. `technical_comments` will have `already_seen` if `already_seen` contains the bot cache